services/src/util/config.rs

```diff
@@ -7,6 +7,14 @@ use serde::Deserialize;
 use snafu::ResultExt;
 use std::path::PathBuf;
 
+pub fn project_root() -> PathBuf {
+    #[cfg(test)]
+    return "../".into();
+
+    #[cfg(not(test))]
+    return ".".into();
+}
```

Comment:
Does this work for the examples? It looks like something we had previously and changed.

Reply:
the examples don't work at all right now because of tokio